server/worker/tasks.py

```diff
@@ -13,6 +13,9 @@
 from ..util.jsonschema import JSONDict
 from .. import config
 
+# Something in the imports above is setting the root logging level to WARNING in
+# production, so we need to explicitly set it back to INFO.
+logging.basicConfig(level=logging.INFO, force=True)
```

Comment:
There's an env var in Heroku called `DD_LOG_LEVEL` that's set to `WARN` for both prod and staging:

<table>
<img width="800" alt="heroku" src="https://github.com/user-attachments/assets/c1cc0319-a7e5-476d-8ed4-84fb4fa29a18">
</table>

Setting it to `INFO` might do the trick and render this change unnecessary?

Reply:
Interesting idea. I tried making that change, but it looks like that just controls the logging of the Datadog agent itself. Also, I was debugging this issue in a review app that didn't have any Datadog integration set up at all, which further makes me think it's not part of the problem.